infra/postdeployment.tf

```
@@ -15,8 +15,20 @@
  */
 
 
-resource "google_compute_network" "default" {
-  name = "default"
+module "gce-vpc" {
+  count        = var.init ? 1 : 0
+  source       = "terraform-google-modules/network/google"
+  version      = "~> 6.0"
+  project_id   = var.project_id
+  network_name = "gce-init-network"
+
+  subnets = [
+    {
+      subnet_name   = "subnet-gce-int"
+      subnet_ip     = "10.10.10.0/24"
+      subnet_region = var.region
+    }
+  ]
```

Needs to depend on the service being enabled (I had errors about this)
```suggestion
  ]
  
  depends_on = [google_project_service.enabled]
```